src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py

```diff
@@ -1070,25 +1070,38 @@ def get_enable_ultra_ssd(self) -> bool:
         # this parameter does not need validation
         return enable_ultra_ssd
 
+    # Mutable Fips now allows changes after create
     def get_enable_fips_image(self) -> bool:
```

Comment:
looks like worth adding some ut

Reply:
tests have been added under "test_agentpool_decorator.py" and "test_aks_commands.py"